x-pack/plugins/siem/common/detection_engine/schemas/response/rules_schema.ts

```diff
@@ -155,30 +153,11 @@ export const rulesSchema = new t.Type<
   'RulesSchema',
   (input: unknown): input is RulesWithoutTypeDependentsSchema => isObject(input),
   (input): Either<t.Errors, RulesWithoutTypeDependentsSchema> => {
-    const output = checkTypeDependents(input);
-    if (!hasListsFeature()) {
-      // TODO: (LIST-FEATURE) Remove this after the lists feature is an accepted feature for a particular release
-      return removeList(output);
```

Comment:
NOTE: This was just a nice to have when switching between between having lists and not having lists. This shouldn't break anything by removing the feature flag here other than you have to do a:

```ts
./hard_reset
```

If you're moving from having lists and not having lists again.

Reply:
Just clarifying for myself: one would have to run the `hard_reset` script within the detection_engine scripts (we've got multiple `hard_reset`s now) in order to remove rules documents that include lists.

Reply:
Correct, this was a bit of forwards and backwards compatibility. A side effect though is that even with all your feature flags turned off it will return `exceptions_list: []` as an empty array but the front end does not validate and reject so that shouldn't hurt anything.